test_dir_1/new_file

```diff
@@ -0,0 +1 @@
+now this is under test_dir_1
```

Comment:
A commit was made on test_dir_1/new_file in RiflerRick/BB_MCS_Test authored by @RiflerRick